README.md

```diff
@@ -94,6 +94,20 @@ You can adjust the order of tabs on this screen too.
 - Tap the hashtag in a post to view all posts with this hashtag
 - Tap the "mute" icon at the top right
 
+### How do I translate a post?
+
+> This feature is available in Tusky 25 or above and Nightly builds from 9th March 2024 or later. To check the Tusky version you are using open the "About" menu.
+
+- Open the three-dot-menu on any post not in your display language. Choose "translate".
+
+## Why can't I see the "translate" option on posts?
+
+Tusky supports translating posts via the connected Mastodon server since version 25. If the option doesn't show up it can have several reasons:
+
+- You are not using Tusky 25 or later
+- Your server does not support translation. If you also can't translate any posts in the web interface, this is the most likely reason. (Tusky uses the `configuration.translation.enabled` attribute of the `/api/v2/instance` endpoint to detect server support.)
```

Comment:
Maybe explicitly say that they can check by browsing to `https://my.instance/api/v2/instance`?